Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -224,3 +192,29 @@ private func cookieFrom(refTag: RefTag, project: Project) -> HTTPCookie? {
 
   return HTTPCookie(properties: properties)
 }
+
+private func fetchProjectAndLiveStreams(projectOrParam: Either<Project, Param>)
+  -> SignalProducer<(Project, [LiveStreamEvent]?), NoError> {
+
+    let param = projectOrParam.ifLeft({ Param.id($0.id) }, ifRight: id)
+
+    let projectAndLiveStreams = AppEnvironment.current.apiService.fetchProject(param: param)
+      .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+      .demoteErrors()
+      .flatMap { project -> SignalProducer<(Project, [LiveStreamEvent]?), NoError> in
```

Comment:
I also think the call-site being a `switchMap` should take care of it but perhaps @mbrandonw can confirm.